exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/AbstractGrpcTelemetryExporterTest.java

```diff
@@ -988,6 +988,7 @@ void toBuilderEquality()
         assertThat(copy.unwrap())
             .extracting("builder")
             .usingRecursiveComparison()
+            .withStrictTypeChecking()
```

Comment:
here's the Javadoc for this method: https://javadoc.io/static/org.assertj/assertj-core/3.24.2/org/assertj/core/api/RecursiveComparisonAssert.html#withStrictTypeChecking()